public/index.html

```diff
@@ -1,28 +1,108 @@
 <!doctype html>
 <html>
 <head>
-	<title>Socket.IO chat</title>
-	<style>
-		* { margin: 0; padding: 0; box-sizing: border-box; }
-		body { font: 13px Helvetica, Arial; }
-		form { background: #000; padding: 3px; position: fixed; bottom: 0; width: 100%; }
-		form input { border: 0; padding: 10px; width: 90%; margin-right: .5%; }
-		form button { width: 9%; background: rgb(130, 224, 255); border: none; padding: 10px; }
-		#messages { list-style-type: none; margin: 0; padding: 0; }
-		#messages li { padding: 5px 10px; }
-		#messages li:nth-child(odd) { background: #eee; }
-	</style>
+    <title>Socket.IO chat</title>
+    <style>
+        * {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+        }
+
+        body {
+            font: 13px Helvetica, Arial;
+        }
+
+        form {
+            background: #000;
+            padding: 3px;
+            position: fixed;
+            bottom: 0;
+            width: 100%;
+        }
+
+        form input {
+            border: 0;
+            padding: 10px;
+            width: 90%;
+            margin-right: .5%;
+        }
+
+        form button {
+            width: 9%;
+            background: rgb(130, 224, 255);
+            border: none;
+            padding: 10px;
+        }
+
+        #messages {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        #messages li {
+            padding: 5px 10px;
+        }
+
+        #messages li:nth-child(odd) {
+            background: #eee;
+        }
+
+        #messages {
+            margin-bottom: 40px
+        }
+    </style>
 </head>
 <body>
 <ul id="messages"></ul>
 <form action="">
-	<input id="m" autocomplete="off" /><button>Send</button>
+    <input id="myName" autocomplete="off" placeholder="Your Name"/>
+    <br>
+    <br>
+    <br>
+    <input id="myText" autocomplete="off" placeholder="Text"/>
+    <button>Send</button>
 </form>
-
 <script src="/socket.io/socket.io.js"></script>
+<script src="https://code.jquery.com/jquery-1.11.1.js"></script>
 <script>
-	var socket = io();
-</script>
+    $(function () {
+        var socket = io();
+        $('form').submit(function () {
+
+            var msg = $('#myText').val();
+
+            // if the user entered command
```

Comment:
recommended: for better readability, use functions instead of comments.
for example, in this case, create ``isMessageCommand(message)`` function instead of writing comments.